### PR TITLE
fix: counterparty not showing in rule conflict

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [x] Fix ounterparty uses wrong data.
- [x] Fix ENS not showing when first adding accounts.

### CODE REVIEW

File: frontend/app/src/composables/blockchain/index.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export const useBlockchains = () => {
     periodic = false
   ) => {
     const chain = get(blockchain);
-    await fetchAccounts(chain);
+    await fetchAccounts(chain, true);
```
Comment (kelsos) [inline frontend/app/src/composables/blockchain/index.ts:69]: do we only call the method manually?

File: frontend/app/src/components/helper/DataTable.vue [hunk 2]
```diff
@@ -349,27 +349,25 @@ const { dark } = useTheme();
...
 
-      &:hover {
-        td {
-          @apply bg-rui-grey-200;
+      &:not(.v-data-table__expanded__content) {
+        &:hover {
+          td {
+            @apply bg-rui-grey-200;
+          }
         }
```
Comment (lukicenturi) [inline frontend/app/src/components/helper/DataTable.vue:357]: sorry, small addition, need to add a little exception for this

File: frontend/app/src/composables/blockchain/index.ts (hunk 1 above)
Comment (lukicenturi) [reply frontend/app/src/composables/blockchain/index.ts:69]: for the refreshAccounts, no, sometimes it automatically, or can be triggered by refresh button.
Maybe I should rename it from `userInitiated` cause the purpose only to refresh the ens

Comment (lukicenturi) [reply frontend/app/src/composables/blockchain/index.ts:69]: Okay, updated

**Pull request merged**